### PR TITLE
Adds fragile melee flag to fake weapons and chainsaws

### PR DESCRIPTION
**Pull request by Brian-Otten**
#### Summary

```SUMMARY: Balance "Adds fragile melee flag to fake weapons and chainsaws"```

#### Purpose of change
Weapons that you'd expect to be fragile aren't fragile.

#### Describe the solution
Add the fragile flag to melee weapons that would break rapidly if used in combat. I didn't do a full audit of the items so this pass was the obvious offenders, fake medieval weapons and activated chainsaws.

### CODE REVIEW

**Comment by Zireael07** [comment]
Fake weapons are metallic, not plastic, so I don't think they'd be fragile. Chainsaws, though, good call.

**Comment by Brian-Otten** [comment]
When i think of the fake weapons i think of something like this:
https://twitter.com/klara_sjo/status/1252373946730569729
Flimsy and fragile weapons that only meet the bare minimum requirements to be similar in appearance. Just being "made of metal" won't make them sturdy.
For weapons that are slightly better fakes we have all the inferior weapons, which have somewhat higher stats and are not as fragile.

**Comment by kevingranade** [comment]
Yea I agree, "fake" is intended to be "not actually a credible example of this kind of weapon", wheres "inferior" is "a substantially substandard example of this kind of weapon".
Fragile definitely fits for "fake".

File: data/json/items/melee/bludgeons.json [hunk 1]
```diff
@@ -227,7 +227,7 @@
     "price": 12500,
     "price_postapoc": 500,
     "material": "wood",
-    "techniques": [ "RAPID", "WBLOCK_2", "SWEEP" ],
+    "techniques": [ "RAPID", "WBLOCK_2", "SWEEP", "FRAGILE_MELEE" ],
     "flags": [ "SHEATH_SWORD" ],
```
Comment (kevingranade) [inline data/json/items/melee/bludgeons.json:231]:
```suggestion
    "techniques": [ "RAPID", "WBLOCK_2", "SWEEP" ],
    "flags": [ "SHEATH_SWORD", "FRAGILE_MELEE" ],
```

File: data/json/items/melee/bludgeons.json [hunk 2]
```diff
@@ -940,7 +940,7 @@
     "description": "A fake shillelagh massproduced as a souvenir for tourists.  This knotty stick has not been cured in a chimney like a traditional shillelagh but has had fake black soot painted on.",
     "price": 2500,
     "material": "wood",
-    "techniques": [ "WBLOCK_1", "SWEEP" ],
+    "techniques": [ "WBLOCK_1", "SWEEP", "FRAGILE_MELEE" ],
```
Comment (kevingranade) [inline data/json/items/melee/bludgeons.json:943]:
```suggestion
    "techniques": [ "WBLOCK_1", "SWEEP" ],
    "flags": [ "FRAGILE_MELEE" ],
```

File: data/json/items/melee/swords_and_blades.json [hunk 3]
```diff
@@ -511,7 +511,7 @@
     "price": 13500,
     "price_postapoc": 50,
     "material": "aluminum",
-    "techniques": [ "WBLOCK_2", "PRECISE" ],
+    "techniques": [ "WBLOCK_2", "PRECISE", "FRAGILE_MELEE" ],
     "flags": [ "SHEATH_SWORD" ],
```
Comment (kevingranade) [inline data/json/items/melee/swords_and_blades.json:515]:
```suggestion
    "techniques": [ "WBLOCK_2", "PRECISE" ],
    "flags": [ "SHEATH_SWORD", "FRAGILE_MELEE" ],
```

File: data/json/items/melee/swords_and_blades.json [hunk 4]
```diff
@@ -1262,7 +1262,7 @@
     "price": 5000,
     "price_postapoc": 50,
     "material": "aluminum",
-    "techniques": [ "RAPID", "WBLOCK_2" ],
+    "techniques": [ "RAPID", "WBLOCK_2", "FRAGILE_MELEE" ],
     "flags": [ "SHEATH_SWORD" ],
```
Comment (kevingranade) [inline data/json/items/melee/swords_and_blades.json:1266]:
```suggestion
    "techniques": [ "RAPID", "WBLOCK_2" ],
    "flags": [ "SHEATH_SWORD", "FRAGILE_MELEE" ],
```

**Comment by Brian-Otten** [comment]
Oh man i really need to stop doing these PR's when tired, those were some pretty stupid mistakes. Thanks for the review.

**Pull request merged**